src/Libraries/RevitNodesUI/Selection.cs

```diff
@@ -581,6 +599,13 @@ public DSFaceSelection()
                 SelectionObjectType.Face,
                 "Select a face.",
                 "Face of Element Id") { }
+        
+        [JsonConstructor]
```

Comment:
so you'll need to add these for all derived selection types right?

Reply:
yes I believe so, I only did this first one as an example so far.  I am still figuring out why I can't modify the selection once reopening the json file before I add them all.